invoice.go

```diff
@@ -317,28 +334,37 @@ type InvoiceList struct {
 	Data []*Invoice `json:"data"`
 }
 
+// InvoiceLineDiscountAmount represents the amount of discount calculated per discount for this line item.
+type InvoiceLineDiscountAmount struct {
+	Amount   int64     `json:"amount"`
+	Discount *Discount `json:"discount"`
+}
+
 // InvoiceLine is the resource representing a Stripe invoice line item.
 // For more details see https://stripe.com/docs/api#invoice_line_item_object.
 type InvoiceLine struct {
-	Amount           int64               `json:"amount"`
-	Currency         Currency            `json:"currency"`
-	Description      string              `json:"description"`
-	Discountable     bool                `json:"discountable"`
-	ID               string              `json:"id"`
-	InvoiceItem      string              `json:"invoice_item"`
-	Livemode         bool                `json:"livemode"`
-	Metadata         map[string]string   `json:"metadata"`
-	Period           *Period             `json:"period"`
-	Plan             *Plan               `json:"plan"`
-	Price            *Price              `json:"price"`
-	Proration        bool                `json:"proration"`
-	Quantity         int64               `json:"quantity"`
-	Subscription     string              `json:"subscription"`
-	SubscriptionItem string              `json:"subscription_item"`
-	TaxAmounts       []*InvoiceTaxAmount `json:"tax_amounts"`
-	TaxRates         []*TaxRate          `json:"tax_rates"`
-	Type             InvoiceLineType     `json:"type"`
-	UnifiedProration bool                `json:"unified_proration"`
+	Amount           int64                        `json:"amount"`
+	Currency         Currency                     `json:"currency"`
+	Description      string                       `json:"description"`
+	Discountable     bool                         `json:"discountable"`
+	Discounts        []*Discount                  `json:"discounts"`
+	DiscountAmounts  []*InvoiceLineDiscountAmount `json:"discount_amounts"`
```

Comment:
Isn't this called "total_discount_amounts"?

Reply:
oh no I'm wrong, didn't realize where I was

Reply:
yeah stripe-go kind of groups nested resources together and I always end up losing myself in those files 😓